v1Tov2Migrations/helper/createTree.js

```diff
@@ -31,8 +31,8 @@ const createTree = async (capture, trx, captureTags) => {
     ),
     attributes: capture.attributes,
     species_id: capture.species_id,
-    created_at: new Date().toISOString(),
-    updated_at: new Date().toISOString(),
+    created_at: capture.captured_at,
```

Comment:
@Kpoke now I remember the things, what I need is set the tree created_at to the captured at time, because in the matching tool, I need this time to compare to the capture's time to make sure the time range is okay: https://github.com/Greenstand/treetracker-api/blob/2e12774e45c71c9abef64b135bd339a768b5cfd5/database/migrations/sqls/20230501015815-materialized-view-capture-match-up.sql#LL18C75-L18C75

Reply:
I have moved the change to capture.create_at, I will keep the change here, to generate the data.

Reply:
I think we can have a new field on the tree for `captured_at`. I think we should have some sort of timestamp that will keep track of when the tree was created in the database

Reply:
I think we don't need it, but in your code to deploy to airflow, you can keep: `created_at: new Date().toISOString()` which means the creation time of this new tree.
The reason I use `created_at: capture.captured_at` is that I am manually creating a huge number of tree and they would got the same time if I use new Date, it make no sense, right?
And captured_at is confusing in `tree`, right? actually that's the time of the first capture.

Reply:
Then maybe we can have `first_capture_captured_at`. I just think we should separate it and not name it `created_at`. It is easy to misunderstand it as when the tree was actually created in the database

Reply:
@Kpoke how about we discuss this in the production meeting?